stories/components/MenuItem/MenuItemStyled.tsx

```diff
@@ -1,17 +1,23 @@
 import styled from 'styled-components';
 
-type MenuItemProps = {
-  active: boolean;
+type props = {
```

Comment:
```suggestion
type MenuItemProps = {
```
Sao chỗ này lại chuyển về props rồi